man/systemd.link.xml

```diff
@@ -443,6 +443,129 @@
         </listitem>
       </varlistentry>
       <varlistentry>
+        <term><varname>Advertise=</varname></term>
+        <listitem>
+          <para>The advertise option is used to set what speeds and modes of operation
+          (duplex mode) are advertised for auto-negotiation. The argument is one or more
+          hexadecimal values from below table.
+
+          <table>
+            <title>Supported advertise values</title>
+
+            <tgroup cols='2'>
+              <colspec colname='advertise' />
+              <colspec colname='explanation' />
+              <thead><row>
+                <entry>value</entry>
+                <entry>Description</entry>
+              </row></thead>
+              <tbody>
+                <row><entry><varname>0x001</varname></entry>
+                <entry>10baseT Half</entry></row>
```

Comment:
This is not user friendly. We really should translate these to friendly names, i.e. instead of expecting "0x001" or so, introduce names such as "10baset-half" or so, that users can use.